package.json

```diff
@@ -7,8 +7,8 @@
         "lint": "npm run lint:lwc && npm run lint:aura",
         "lint:lwc": "eslint **/lwc/**",
         "lint:aura": "sfdx force:lightning:lint **/aura/** --exit",
-        "prettier": "prettier --write '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml}'",
-        "prettier:verify": "prettier --list-different '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml}'",
+        "prettier": "prettier --write '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml,cls}'",
```

Comment:
Add formatting for triggers to all three occasions in the `package.json`. And it would be great if we could stick with alphabetical order of file extension names. ;-)

Reply:
Updated the file with trigger support, and reordered the extensions alphabetically.